src/__tests__/Headertest..tsx

```diff
@@ -4,11 +4,10 @@
 
 import React from 'react'
 import 'react-native'
-
 // Note: test renderer must be required after react-native.
 import renderer from 'react-test-renderer'
-import { App } from '../src/App'
```

Comment:
There should be test for the app but for the time being they have been removed because they did not work with the navigation.